INFO 2021-06-04 16:40:07,744 remote_connection.py:117 __init__ Could not connect to port 4723 on host localhost
INFO 2021-06-04 16:40:07,745 remote_connection.py:136 __init__ Could not get IP address for host: localhost
DEBUG 2021-06-04 16:40:07,747 remote_connection.py:388 _request POST http://localhost:4723/wd/hub/session {"capabilities": {"firstMatch": [{"platformName": "Android", "appium:platformVersion": "10", "appium:deviceName": "89LJQWROR47TCYKJ", "appium:appPackage": "com.ss.android.ugc.aweme", "appium:appActivity": ".splash.SplashActivity", "appium:unicodeKeyboard": false, "appium:resetKeyboard": false, "appium:noReset": true, "appium:newCommandTimeout": 6000, "appium:automationName": "UiAutomator2"}]}, "desiredCapabilities": {"platformName": "Android", "platformVersion": "10", "deviceName": "89LJQWROR47TCYKJ", "appPackage": "com.ss.android.ugc.aweme", "appActivity": ".splash.SplashActivity", "unicodeKeyboard": false, "resetKeyboard": false, "noReset": true, "newCommandTimeout": 6000, "automationName": "UiAutomator2"}}
DEBUG 2021-06-04 16:40:07,758 connectionpool.py:231 _new_conn Starting new HTTP connection (1): localhost:4723
